static/scripts/template-script.js

```diff
@@ -0,0 +1,53 @@
+const burgerButton = document.querySelector('.burger-btn');
+const navBarContent = document.querySelector('.navbar-content');
+const navBarLinks = document.querySelectorAll('.navbar-links');
+const content = document.querySelector('.container');
+const dropdownMenu = document?.querySelector('.dropdown');
+const dropDownContent = document?.querySelector('.dropdown-content');
+
+// show/hide navbar content
+burgerButton?.addEventListener('click', () => {
```

Comment:
burger?

Reply:
hamburger* :)

Reply:
So we keep it with this name?

Reply:
yep why not ? it's right name for that thing
google burger/hambuger menu css